packages/brokeneck-react/src/test-utils/mockRootContext.js

```diff
@@ -0,0 +1,918 @@
+export default {
```

Comment:
do we really need this really big mock here? if we're not using most of the stuff in here, as I expect, can we keep it smaller and limited to only what we need?

Reply:
I will reduce it to only the schema types we use in the current tests, but I guess we'll have to keep adding the missing ones when they are needed.